### ISSUE TITLE
Is full inter-character spacing common for narrow columns in Tamil?

### ISSUE THREAD

**Issue by r12a** [w3c/iip#58]
When only a single word fitted in a column of an edition of the Tamil Murasu newspaper, intercharacter spacing was applied to stretch the word to the width of the column. Equal spacing was applied between each glyph, regardless of whether or not it was a vowel-sign, as long as the vowel-sign glyphs were not connected with the base character. Large gaps were still tolerated, however, if only 2 short words fitted on the line. Vowel-signs that are not already slightly separated from their base are not letter-spaced. For example the i in கிராமப்புறங்கலில். Ligated forms are also not separated.

See https://github.com/w3c/type-samples/issues/76

Is this a common practise?  @murasu

**Comment by murasu** [comment]
Yes, this is common practice. Attached are portions of a printed Tamil newspaper in Malaysia (Tamil Malar) that shows the same practice.
![TamilMalar-20191205-2](https://user-images.githubusercontent.com/22926756/70193196-5d0e1480-1739-11ea-8143-7513c07658dd.jpg)
![TamilMalar-20191205-1](https://user-images.githubusercontent.com/22926756/70193198-5d0e1480-1739-11ea-9172-3e9546392a35.jpg)

**Comment by r12a** [comment]
Thanks, Muthu. There are some interesting differences in your scan from the one i posted earlier – particularly in the bottom example. Though nothing surprising.

1. In several places, such as about half-way down the right column, the text appears to have been compacted in order to fit extra words on a line.
2. In several places, such as bottom para in middle column, intra-word spacing is expanded when there are two words on a line (not just one as in the scan i pointed to).
3. in the same column the letter-spacing separates the d and u vowel-sign in ஜு.

**Comment by murasu** [comment]
These are workarounds applied to overcome the absence of proper word-breaking features in the layout software. I believe in this case it is InDesign.

**Comment by r12a** [comment]
I added something about this to the gap analysis doc – please review.
https://w3c.github.io/iip/gap-analysis/taml-gap.html#syllable_spacing

I also created an [interactive test](http://w3c.github.io/i18n-tests/css-text/interactive/int-text-justify-000.html) and a [test script](https://github.com/w3c/i18n-tests/wiki/Test-script-for-text-justify) related to justification.
